src/utils/shortId.ts

```diff
@@ -0,0 +1,4 @@
+/**This is needed, because we want to reduce the id/MAC-adress displayed for the
+ * robots to a given number that should be equal everywhere
+ */
```

Comment:
```suggestion
/**
 * This is needed, because we want to reduce the id/MAC-adress displayed for the
 * robots to a given number that should be equal everywhere
 */
```